README.md

```diff
@@ -68,23 +68,28 @@ class A
 end
 
 # a_spec.rb
-require 'a'
+require "a"
+require "interjectable/rspec_test_helper"
```

Comment:
WDYT of renaming the file so it's just `require "interjectable/rspec"`?

Reply:
Sure.